pkg/vpcmodel/routing_paths.go

```diff
@@ -44,6 +46,37 @@ func (p Path) Empty() bool {
 	return len(p) == 0
 }
 
+// ConcatPaths returns a new path concatenating the passed in paths
+func ConcatPaths(paths ...Path) Path {
+	return slices.Concat(paths...)
+}
+
+func ConcatWithResource(resource VPCResourceIntf, path Path) Path {
+	return ConcatPaths(PathFromResource(resource), path)
+}
+
+func PathFromResource(resource VPCResourceIntf) Path {
+	return []*Endpoint{{VpcResource: resource}}
+}
+
+func PathFromNextHopValues(nextHop, origDest string) Path {
+	n, _ := ipblock.FromCidrOrAddress(nextHop)
+	o, _ := ipblock.FromCidrOrAddress(origDest)
+	return []*Endpoint{{NextHop: &NextHopEntry{NextHop: n, OrigDest: o}}}
+}
```

Comment:
Errors are not being checked in an exported function. Better move to `pkg/ibm/vpc/ingress_routing_test.go`

Reply:
done